Created randomization test and added it to the suite

mondiaz opened the pull request:
I created a randomized problem test case now, which checks to make sure
the question’s variables are changing, and added it to the test suite.

mondiaz commented:
@gbruhns can you review this tomorrow / later this week?

gbruhns commented:
This test is sometimes failing for me on the last verifyEval, but it is not consistently reproducible. Seems like it might be because the variables are not changing? Not sure how to make sure they always change when you reset.

mondiaz commented:
I looked into this further and made a few improvements.

1. The randrange values were a bit limited (n1 = (5,89,19); n2 = (2,10,5)) which meant the only numbers that would ever come up for n1 were 5,24,43,62 and 81, and for n2, 2,7,12.  I increased the range.
2. The randomization setting seems to allow for two problem values in a row being identical every umpteenth time you reset the problem.  To try and account for that behavior, I added a check for a third variable and am only ensuring that the problem values in variables 1 and 3 don't match. 
3. I also added an echo to print the variables' problem text / values in the log for easier debugging. Looks like this: [info] echo: What is 404 divided by 172?
4. I also added a focus command to scroll to where the variables are changing on the page.

Could you test again @gbruhns ?
